2017-04/april-08.md

```diff
@@ -0,0 +1,83 @@
+## April 8 ([discuss](https://github.com/babel/notes/pull/19))
+
+### Babel 7 Updates
+
+- [Upgrade to Babel 7](https://github.com/babel/babel.github.io/pull/1146): We posted this for the alpha version already (v7.0.0-alpha.7), which we don't make any guarantees on (although we are using it at Behance).
```

Comment:
`which we don't make any guarantees on` -> `which we don't make any guarantees about`